src/style.css

```diff
@@ -0,0 +1,124 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap);
+
+* {
+  margin: 0;
+  font-family: Roboto , sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+
+.apple {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-decoration: none;
+  padding: 0 16px;
+  margin-top: 40px;
+}
+
+.code {
+  display: flex;
+  align-items: flex-start;
+  width: 166px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.block {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  gap: 5px;
+}
+  
+.stars .stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Reuse stars loop from previous work. Code should work for all stars if I'll change star modifier in devtools